backends-common/elasticsearch-v8/src/main/java/org/apache/james/backends/es/v8/ElasticSearchIndexer.java

```diff
@@ -92,7 +92,7 @@ public Mono<BulkResponse> update(List<UpdatedRepresentation> updatedDocumentPart
         return client.bulk(bulkBuilder.build())
             .onErrorResume(exception -> {
                 LOGGER.warn("Error while updating index", exception);
-                return Mono.empty();
+                return Mono.error(exception);
```

Comment:
Log then rethrow is an anti-pattern...

Here we likely should not do anything I guess.

Reply:
Same remark